.htaccess

```diff
@@ -30,12 +30,11 @@ DirectoryIndex index.php index.html index.htm
 
 # PHP 5, Apache 1 and 2.
```

Comment:
Let's delete this code comment and the empty line above it. Drupal used to include PHP 4 equivalents, but without them here, this code comment is superfluous.